src/operator/toPromise.ts

```diff
@@ -57,5 +56,5 @@ export function toPromise<T>(this: Observable<T>, PromiseCtor: PromiseConstructo
  * @owner Observable
  */
 export function toPromise<T>(this: Observable<T>, PromiseCtor?: PromiseConstructorLike): Promise<T> {
```

Comment:
can't we now simplify signature as 
```
const toPromise : typeof Observable.prototype.toPromise = ...
```
and remove duplicated signature? (not sure this'll work current version of TSC we're using though)